### ISSUE TITLE
Button to edit own user

### ISSUE THREAD

**Issue by hendrix15** [hpi-swt2/compass-portal-blue#128]
As a user, I want to be able to change my own user profile by clicking on a button on the main page.

**Comment by TheGrayStone** [comment]
This most probably depends on #127

**Comment by hendrix15** [comment]
![layout search](https://user-images.githubusercontent.com/45318704/148410536-f2eccb08-3530-4d69-a83f-43991259055e.jpg)

**Comment by hendrix15** [comment]
isn't this already done?

**Comment by Olliwehr** [comment]
> isn't this already done?

@hendrix15 ahh yeah, you are right. 👍 
It is definitively part of PR #153. So it was probably just forgotten to be related to and closed by the PR.
